app/views/curation_concerns/file_sets/show.html.erb

```diff
@@ -11,14 +11,13 @@
 <div class="container-fluid">
   <div class="row">
     <div class="col-xs-12 col-sm-4">
-      <%= render 'generic_files/media_display' %>
+      <%= media_display @presenter %>
       <%= render 'show_actions' %>
       <%= render 'show_collections' %>
     </div>
     <div itemscope itemtype="<%= @presenter.itemtype %>" class="col-xs-12 col-sm-8">
       <h1 class="visibility"><%= display_multiple @presenter.title %> <%= render_visibility_badge %></h1>
       <p class="genericfile_description"><%= display_multiple @presenter.description %></p>
-      <%= render 'show_descriptions' %>
```

Comment:
Are you deleting this because there's currently no way to edit the descriptions that would be shown here? If so, I'd say we should use `<%# TODO: render 'show_descriptions' %>` instead, since there are discussions about allowing FileSet descriptions in Sufia 7. (If not in the 7.0.0 release.)